tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -347,6 +347,34 @@ def test_pantsd_pid_change(self):
       # Remove the pidfile so that the teardown script doesn't try to kill process 9.
       os.unlink(pidpath)
 
+  def test_pantsd_memory_usage(self):
+    """Validates that after N runs, memory usage has increased by no more than X percent."""
+    number_of_runs = 10
+    max_memory_increase_fraction = 0.4
+    with self.pantsd_successful_run_context() as (pantsd_run, checker, workdir, config):
+      cmd = ['list', 'testprojects::']
```

Comment:
I'm not sure, but it may happen that some target in `testprojects` does not parse (this is why, for instance, we ignore `.*/testprojects/.*` in the contrib section of `build-support/bin/ci.sh`). Maybe this would better be `examples`?

Reply:
Everything in the repo is parseable, although it is not all buildable.